explainer.md

```diff
@@ -279,21 +293,19 @@ CSS can be used to make changes to the default animation, or override `animation
 
 ### JavaScript access to elements
 
-So far the transition elements have been addressed by pseudo-element selectors, but JavaScript could be given access to the elements. This would be done via a shadow root.
-
-This isn't unusual, as developer currently use things like [`::placeholder`](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder) to address things in a UA shadow root within `<input>`. We can't expose the shadow root within `<input>`, because the structure is non-standard, however, for transitions, the structure will be standardized, so it can be exposed.
+So far the transition elements have been addressed by pseudo-element selectors, but JavaScript could be given access to the elements using the [CSSPseudoElement](https://drafts.csswg.org/css-pseudo-4/#CSSPseudoElement-interface) interface.
 
 ```js
 // In Page-B
 document.performTransition((transition) => {
   // Build up transition parts, then…
-  transition.root.querySelector('[part=header]').animate(…);
+  document.documentElement.pseudo("page-transition-container(name)").animate(...);
```

Comment:
This API doesn't exist. Is it being proposed here?

Reply:
You mean the .animate API on CSSPseudoElement? Yeah, I was proposing that.

I've noticed a bunch of APIs for pseudo elements work by adding it as an argument, like window.getComputedStyle(element, "::after"). This is likely there wasn't an object in script for a pseudo element when these were added. Now that we have one, why not add these APIs to that interface.